### ISSUE TITLE
Transit feature request

### ISSUE THREAD

**Issue by CarlosBoix** [stoermelder/vcvrack-packone#397]
Hello, would it be possible to add colour coding to the preset buttons please ? Would be ace thank you

**Comment by stoermelder** [comment]
Do you mean a custom color per preset for the button?

**Comment by CarlosBoix** [comment]
yes, could be just 4 like cable colours but more are welcome too

**Comment by stoermelder** [comment]
https://github.com/stoermelder/vcvrack-packone/commit/59a188fa4cef99113237fc9ecb0aa338c043757e
